### DISCUSSION TITLE
Name space

### DISCUSSION THREAD

**Discussion by LiveManRU** [outline/outline#4518] [Ideas]
Hi! There is one of two important things that, to my surprise, is missing from your wonderful project! Namespace - variables that can be used in articles, for example, the name of the service, which may change, and in order not to manually change the name of the service in each article, change the value of the variable, which will display a new value in all articles referring to this variable.

**Comment by tommoor** [comment]
I'm not sure this is very possible with our data model. It's never come up before – which software are you aware of that offers this?

**Reply by LiveManRU** [reply]
This is probably the most popular [example](https://en.wikipedia.org/wiki/Wikipedia:Namespace)

**Reply by LiveManRU** [reply]
And this is the most correct [example](https://www.mediawiki.org/wiki/Manual:Variable/en)
